index.html

```diff
@@ -211,15 +211,17 @@ <h2 class="w-100">Contact me</h2>
                             If you have an application you are interested in developing,
```

Comment:
[ OPTIONAL ] Commit messages should be capitalized. Please read [this](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/) article and apply best practices to write commit messages.